tests/models/big_bird/test_modeling_flax_big_bird.py

```diff
@@ -221,3 +227,15 @@ def check_pt_flax_outputs(self, fx_outputs, pt_outputs, model_class, tol=1e-5, n
             return
         else:
             super().check_pt_flax_outputs(fx_outputs, pt_outputs, model_class, tol, name, attributes)
+
+    @is_pt_flax_cross_test
+    # due to jit compilation flax version has to use jax.random with
+    # provided PRGN key instead of np.random
+    def test_equivalence_flax_to_pt(self):
```

Comment:
Is there any way we could make these tests pass instead of removing them? Or write a new test?

Reply:
@patrickvonplaten I was not able to think about any thing in particular yet. But I will give it a try during a weekend.

Reply:
Good catch @amyeroberts! If we set `train=False` here for the Flax model and `model.eval()` for the PyTorch one then there should be no discrepancy (since we won't use any randomness), worth double checking the parent tester class:
https://github.com/huggingface/transformers/blob/d00997e66c0f4f6e55a1a422d6d3ca8fb1d37aab/tests/test_modeling_flax_common.py#L350

Reply:
Hi @sanchit-gandhi @amyeroberts! I had some time to look at the equivalence tests and either I am missing something really simple or there is some missing part in the pytorch implementation of `BigBird` since setting it up to `eval` mode does not reflect in any change in its behavior corresponding to the random attention. What I mean by that is that I cannot find any part in the code that controls whether to use random attentions or not depending on whether we are in the `eval` or `train` mode. It seems like it is always using random attention no matter the mode. Moreover if I manually overwrite the rand_attn in pytorch implementation with `np.zeros` then the outputs from `flax` and `pytorch` match.

Reply:
@sanchit-gandhi that is a great idea! I will skip the tests for now, open issue for the Pytorch version, fix it and "unskip" equivalence tests with that PR as well.